browser/ui/brave_browser.cc

```diff
@@ -85,7 +85,8 @@ void BraveBrowser::ScheduleUIUpdate(content::WebContents* source,
   // We need to update sidebar UI only when current active tab state is changed.
   if (changed_flags & content::INVALIDATE_TYPE_URL) {
     if (source == tab_strip_model_->GetActiveWebContents()) {
-      if (sidebar_controller_) {
+      // sidebar() can return a nullptr in unit tests.
+      if (sidebar_controller_ && sidebar_controller_->sidebar()) {
```

Comment:
should we use CHECK_IS_TEST here to ensure it's only happening in tests?

Reply:
Added in 9a2478ad33ecf80b8047fdae4494dd71776850d7